### PR TITLE
add script to convert OpenPose JSON format

### PR DESCRIPTION
**Pull request by Chevrefeuille**
As discussed, the script to convert OpenPose JSON format to your GCN format. I might improve it to use command line arguments when I have some time.

### CODE REVIEW

**Comment by yysijie** [comment]
Hi, @Chevrefeuille, thank you for your great work!

File: tools/convert_poses.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    with open(video_list_file, 'w') as f:
+        p = Path(videos_path)
+        for path in p.glob('*.avi'):
+            video_path = str(path)
+            video_name = (video_path.split('/')[-1]).split('.')[0]
+            f.write(video_name + '\n')
+
+if __name__ == "__main__":
+    # folder that contains all the video that have been treated by OpenPose
+    videos_path = '../../myprojectfolder/data/videofolder'
```
Comment (yjxiong) [inline tools/convert_poses.py]: Hi, thanks for your contribution!
Would you please change these paths to configurable arguments of this script? So other users can directly use the script without modifying the code.

The `argparse` module will be handy here. For usage you can refer to Sijie's code here

https://github.com/yysijie/st-gcn/blob/master/tools/ntu_gendata.py#L100

**Comment by yjxiong** [comment]
Merging this, @yysijie please update the script to use command line arguments.

@Chevrefeuille Thanks!

**Comment by Chevrefeuille** [comment]
Sorry, I was a little busy recently... I will update the script soon.

**Comment by Zzoey** [comment]
Just wondering if I have real labels to give, what is the format of the fake_labels.json, Can someone please let me know ?

**Comment by OpenMMLab-Assistant001** [comment]
Hi @Chevrefeuille！

First of all, we want to express our gratitude for your significant PR in the mmskeleton project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）

Thank you again for your contribution❤

**Pull request merged**